Tasks/MavenV3/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -46,6 +46,8 @@
   "loc.input.help.mavenOpts": "Sets the MAVEN_OPTS environment variable, which is used to send command-line arguments to start the JVM. The -Xmx flag specifies the maximum memory available to the JVM.",
   "loc.input.label.mavenFeedAuthenticate": "Authenticate built-in Maven feeds",
   "loc.input.help.mavenFeedAuthenticate": "Automatically authenticate Maven feeds from Azure Artifacts. If built-in Maven feeds are not in use, deselect this option for faster builds.",
+  "loc.input.label.skipEffectivePom": "Skip generating effective POM while authenticating built-in feeds",
+  "loc.input.help.skipEffectivePom": "Authenticate built-in Maven feeds using the POM only, allowing parent POMs in Azure Artifacts/TFS [Package Management] feeds.",
```

Comment:
please change TFS to Azure DevOps Server

Reply:
Fixed